test/unit/points/domain/test-tiers.js

```diff
@@ -0,0 +1,35 @@
+const expect = require('chai').expect
+const Tiers = require('../../../../app/points/domain/tiers')
+const pointsHelper = require('../../../helpers/points-helper')
+const Locations = require('../../../../app/staging/constants/locations')
+
+describe('points/domain/Tiers', function () {
+  it('throws an error when any property is undefined', function () {
+    expect(function () {
+      new Tiers(Locations.COMMUNITY,
+      pointsHelper.getTestTiersObject(),
+      pointsHelper.getTestTiersObject(),
+      pointsHelper.getTestTiersObject(),
+      pointsHelper.getTestTiersObject(),
+      pointsHelper.getTestTiersObject(),
+      undefined,
```

Comment:
yea, this is just from my point of view. Push it to the dev chat for a second opinion. Happy to keep as is and we can take this approach with similar things from here